conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md

```diff
@@ -1,46 +1,25 @@
+<!--
 Thanks for reporting your issue.
-
-
 Please fill out the sections below.
+-->
+Issue:
 
-
-Example (code snippet, shell command, or screenshot):
-
-<details>
-
-
-
-</details>
-
-<br>
-Environment (`conda env export`):
-<br>
-<br>
-
+<br/>
+Environment (<code>conda list</code>):
```

Comment:
So the reason for `conda env export` is this creates a nice format that we can trivially recreate a `conda` environment from locally. AFAIK there is no way to do this with `conda list`. Do you know of such a way?

Reply:
Nope. Main reason for the change was that `conda env export` doesn't give you the channel that a package came from and that is important information.

Reply:
@kalefranz, do you have any suggestions for us?

Reply:
conda list --explicit > somefile.txt

Reply:
1. `conda list --explicit`
2. `conda list --export`
3. `conda env export`

Right now, no One Right Way.  That's just where we are at the moment.

Reply:
1 solves the issue of reproducibly creating environments, but is very unreadable (as it contains full URLs).

2 doesn't seem to have the channel information AFAICT.

3 is ideal IMHO, but hasn't listed the channels for individual packages for some time. ( https://github.com/conda/conda/issues/5706 )